datasets/youtube_caption_corrections/youtube_caption_corrections.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dataset built from <auto-generated, manually corrected> caption pairs of
+YouTube videos with labels capturing the differences between the two."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+url={https://github.com/2dot71mily/youtube_captions_corrections}
+},
+year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+Dataset built from pairs of YouTube captions where both an `auto-generated` and
+a `manually corrected` caption was available for a single specified language.
+This dataset looks only at simple (e.g. ignoring case/punctuation/stop words)
+two-way (e.g. ignoring single-sided insertions) single-token differences between the
+auto-gen and manual transcripts.
+More complex differences in the original repo are not captured here.
+"""
+
+_LICENSE = "MIT License"
+
+_CHANNEL_NAMES = [
+    "3Blue1Brown",
+    "Aurélien_Géron",
+    "Jeremy_Howard",
+    "Luis_Serrano",
+    "TED",
+    "Weights_&_Biases",
+    "nature_video",
+    "Alfredo_Canziani",
+    "DeepMind",
+    "Pieter_Abbeel",
+    "Veritasium",
+    "minutephysics",
+    "stanfordonline",
+]
+_URLS = [
+    f"https://raw.githubusercontent.com/2dot71mily/youtube_captions_corrections/main/data/transcripts/en/postproc_transcripts/{channel_name}.json"
+    for channel_name in _CHANNEL_NAMES
+]
+
+
+class YouTubeCaptionCorrections(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class YoutubeCaptionCorrections(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `youtube_caption_corrections`

You'll need to regenerate the dataset_infos.json file after this change